guide/creating-your-bot/command-deployment.md

```diff
@@ -1,7 +1,7 @@
 # Registering slash commands
 
 ::: tip
-This page assumes you use the same file structure as our [Slash commands](/slash-commands/) section, and the provided are made to function with that setup. Please carefully read that section first so that you can understand the methods used in this section.
+This page assumes you use the same file structure as our [Creating slash commands](/creating-your-bot/slash-commands.html#before-you-continue/) section, and the provided are made to function with that setup. Please carefully read that section first so that you can understand the methods used in this section.
```

Comment:
```suggestion
This page assumes you use the same file structure as our [Creating slash commands](/creating-your-bot/slash-commands.html) section, and the provided are made to function with that setup. Please carefully read that section first so that you can understand the methods used in this section.
```